docs/source/data-packages.rst

```diff
@@ -4,7 +4,7 @@ Making Data Packages
 Combined with the `Conda Package Manger <https://conda.io/docs/>`_, Intake makes it possible to create *data packages*
 which can be installed and upgraded just like software packages.  This offers several advantages:
 
-  * Distributing datasets becomes as easy as installing software
+  * Distributing datasets becomes as easy ``conda install``
```

Comment:
I chuckled when I read the original -- if someone says a task is "as easy as installing software" I'd assume it's very difficult :)